explorer/app/viewModelBuilders/anvil-catalog/common/viewModelBuilders.ts

```diff
@@ -51,14 +58,14 @@ export const buildConsentCodes = (
 
 /**
  * Build props for consortium cell component from the given AnVIL entity.
- * @param anvilCatalogEntity - AnVIL catalog entity.
+ * @param anVILCatalogConsortium - AnVIL catalog Consortium .
```

Comment:
Revert this method `buildConsortium` and all associated comments - all AnVIL entities use this model builder.